.github/workflows/e2e-parallel-destroy.yml

```diff
@@ -25,48 +25,48 @@ jobs:
       matrix:
         include:
           - example_path: examples/analytics/emr-on-eks
-            tenant_name: "emr-eks"
+            tenant_name: 'emr-eks'
           - example_path: examples/analytics/spark-k8s-operator
-            tenant_name: "spark"
+            tenant_name: 'spark'
           - example_path: examples/gitops/argocd
-            tenant_name: "argo"
+            tenant_name: 'argo'
           - example_path: examples/aws-efs-csi-driver
-            tenant_name: "efs"
+            tenant_name: 'efs'
           - example_path: examples/crossplane
-            tenant_name: "crossplane"
+            tenant_name: 'crossplane'
           - example_path: examples/eks-cluster-with-new-vpc
-            tenant_name: "new-vpc"
+            tenant_name: 'new-vpc'
           - example_path: examples/fully-private-eks-cluster
-            tenant_name: "private"
+            tenant_name: 'private'
             deployment_order:
               [
-                "module.e2e_test.module.aws_vpc",
-                "module.e2e_test.module.vpc_endpoint_gateway",
-                "module.e2e_test.module.vpc_endpoints",
-                "module.e2e_test.module.eks_blueprints",
+                'module.e2e_test.module.aws_vpc',
```

Comment:
if every `example/*` is structured the same, we could parameterize a lot of this instead of hardcoding the example names to evaluate

Reply:
if `deployment_order` used, it usually means there's a custom order of modules&/resources which is not same order as `DEFAULT_DEPLOY_ORDER` which is what most examples use